### PR TITLE
Restart electron app on wallet lock

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: apps/desktop-wallet/public/electron.js [hunk 1]
```diff
@@ -312,6 +315,20 @@ app.on('ready', async function () {
...
+  ipcMain.handle('app:restart', () => {
+    app.relaunch()
+    app.exit()
+  })
+
+  ipcMain.handle('app:clearCache', async () => {
+    const session = mainWindow.webContents.session
+
+    await session.clearCache()
+    await session.clearStorageData({
```
Comment (mvaivre) [inline apps/desktop-wallet/public/electron.js:327]: Should we keep this if this doesn't solve our issue? I'm a bit scared of potential unforeseen side effects. WDYT?

**Comment by mvaivre** [comment]
Tested, works well (manual lock, auto lock...).

File: apps/desktop-wallet/public/electron.js (hunk 1 above)
Comment (nop33) [reply apps/desktop-wallet/public/electron.js:327]: ok, we can remove it.

Comment (nop33) [reply apps/desktop-wallet/public/electron.js]: pushed

**Review by mvaivre — APPROVED** [review]
Looks good!

**Pull request merged**